src/lib/libraries/backdrops.json

```diff
@@ -120,6 +120,20 @@
             1
         ]
     },
+    {
+        "name": "Ocean",
+        "md5": "fb907f72b310acc8b95cbf2d2cccabc9.svg",
+        "type": "backdrop",
+        "tags": [
+            "underwater",
+            " ipzy"
```

Comment:
@carljbowman Don't leave a space between tags. E.g. `[tag,another tag,foo,bar]`

Reply:
Remove all spaces between tags. Much easier to read these in the JSON then the 2.0 field editor UI.